pkg/apis/deployment/v1/deployment_upgrade_spec.go

```diff
@@ -21,8 +21,11 @@
 package v1
 
 type DeploymentUpgradeSpec struct {
-	// Flag specify if upgrade should be auto-injected, even if is not required (in case of stuck)
+	// AutoUpgrade flag specifies if upgrade should be auto-injected, even if is not required (in case of stuck)
 	AutoUpgrade bool `json:"autoUpgrade"`
+	// DebugLog flag specifies if containers running upgrade process should print more debugging information.
+	// This applies only to init containers.
+	DebugLog bool `json:"debugLog"`
```

Comment:
Provide default value via doc and regenerate docs

Reply:
done 👍🏻